README.md

```diff
@@ -128,6 +128,109 @@ NOTE: You can leave that static server running in one session while you continue
 to execute `tox` on another session. That server will pick up any new changes to
 the report automatically so you don't have to restart it each time.
 
+Troubleshooting
+----------
+
+Since Kubernetes charms are not supported by the `juju debug-hooks`, the only
+way to intercept the control of code execution is to initialize the non-tty-bound
+debugger session and connect to the session externally.
+
+For this purpose, we chose the [rpdb](https://pypi.org/project/rpdb/), the
+remote Python debugger based on pdb.
+
+So, given you have already deployed your model and you would like to debug the
+`config-changed` hook, you should do the following:
+
+Given `prometheus` is an application name and `lma` is a Juju model name, 
+execute the following command to grab an interactive Bash shell to the operator
+pod:
```

Comment:
For example, given that you have already deployed an application named `prometheus` in a Juju model named `lma` and you would like to debug your `config-changed` handler, execute the following: